scripts/generate-ssl-certs.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
+
+if [ ! -e server.js ]
+then
+	echo "Error: could not find main application server.js file"
+	echo "You should run the generate-ssl-certs.sh script from the main MEAN application root directory"
+	echo "i.e: bash scripts/generate-ssl-certs.sh"
+	exit -1
+fi
+
 echo "Generating self-signed certificates..."
+mkdir -p ./config/sslcerts
 openssl genrsa -out ./config/sslcerts/key.pem -aes256 1024
 openssl req -new -key ./config/sslcerts/key.pem -out ./config/sslcerts/csr.pem
 openssl x509 -req -days 9999 -in ./config/sslcerts/csr.pem -signkey ./config/sslcerts/key.pem -out ./config/sslcerts/cert.pem
 rm ./config/sslcerts/csr.pem
-# resolve issue with bad password...
-# Error: error:0906A068:PEM routines:PEM_do_header:bad password read
```

Comment:
@jloveland yes, please open an issue on that and assign it to me, I'll get to it later this week.